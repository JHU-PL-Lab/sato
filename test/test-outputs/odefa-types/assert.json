[
  {
    "err_errors": [
      [
        "Error_value",
        { "err_value_aliases": [ "b", "a" ], "err_value_val": "false" }
      ]
    ],
    "err_input_seq": [],
    "err_location": "res = b ? (ret = 0) : (ab = abort)",
    "err_steps": 62
  }
]