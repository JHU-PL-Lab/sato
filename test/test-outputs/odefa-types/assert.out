Type errors for:
- Input sequence  : []
- Found at clause : res = b ? (ret = 0) : (ab = abort)
--------------------
* Value : b = a = false
1 errors found. No further control flows exist.
